arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,578 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagementClient",
+    "version": "2015-10-31"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+	"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}": {
+      "patch": {
+        "tags": [
+          "AutomationAccounts"
+        ],
+        "operationId": "AutomationAccounts_Update",
+        "description": "Patch an automation account.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/automationaccountoperations"
```

Comment:
The url `http://aka.ms/azureautomationsdk/automationaccountoperations` isn't currently valid -- are you expecting this to be created later?